go.mod

```diff
@@ -216,3 +216,5 @@ require (
 )
 
 go 1.22
+
+replace github.com/hashicorp/raft => github.com/weaviate/raft v0.0.0-20241210082714-bfb17ceb03db
```

Comment:
This uses weaviate's raft fork. With support to avoid duplicate legacy metrics. If our patch landed in upstream. We can just remove this line.

Reply:
instead of replace statement just use `github.com/weaviate/raft v0.0.0-20241210082714-bfb17ceb03db` directly in go.mod instead of `github.com/hashicorp/raft`

Reply:
oh, didn't read it thoroughly, yeah, please ignore my comment and leave this as it is